data/load-seed-data.js

```diff
@@ -10,23 +10,24 @@ async function run() {
   try {
 
     await Promise.all(
-      cats.map(cat => {
+      characters.map(character => {
         return client.query(`
-          INSERT INTO cats (name, type, url, year, lives, is_sidekick)
+          INSERT INTO mortal_kombat (name, species, url, introduced, is_ninja, fighting_style)
           VALUES ($1, $2, $3, $4, $5, $6);
         `,
-        [cat.name, cat.type, cat.url, cat.year, cat.lives, cat.isSidekick]);
+          // eslint-disable-next-line indent
```

Comment:
this line is failing your tests but they look like they should pass, so I'm going to give you points.